cmd/nfs-subdir-external-provisioner/provisioner.go

```diff
@@ -110,11 +110,33 @@ func (p *nfsProvisioner) Provision(ctx context.Context, options controller.Provi
 		}
 	}
 
-	glog.V(4).Infof("creating path %s", fullPath)
-	if err := os.MkdirAll(fullPath, 0777); err != nil {
+	createMode := os.FileMode(0777)
+	annotationCreateMode, exists := metadata.annotations["nfs.io/createMode"]
+    if exists {
+        annotationCreateModeUInt, _ := strconv.ParseUint(annotationCreateMode, 8, 32)
+        createMode = os.FileMode(annotationCreateModeUInt)
+    }
+
+    createUID := "0"
```

Comment:
root:root is really the default everywhere?

Reply:
it looks like the previous pull request behavior